### PR TITLE
add notes for 1559 implementer's call #7

### PR DESCRIPTION
**Pull request by gichiba**
related to #226

### CODE REVIEW

**Comment by gichiba** [comment]
#222 should probably be merged before this one

File: Fee Market Meetings/Meeting 07.md [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+- Baranbe Monnot
+- Micah Zoltu
+- Ramil Amerzyanov
+- Rai Sur
+- Rick Dudley
+- Tomasz
+
+### Next Meeting
+
+TBD
```
Comment (timbeiko) [inline Fee Market Meetings/Meeting 07.md:724]: Next meeting has been set for December 17, 16:00 UTC

**Review by timbeiko — APPROVED** [review]
One tiny comment, but approved. Will merge after the previous call's notes are in :+1:

File: Fee Market Meetings/Meeting 07.md (hunk 1 above)
Comment (timbeiko) [reply Fee Market Meetings/Meeting 07.md:724]:
```suggestion
December 17, 16:00 UTC ([Agenda](https://github.com/ethereum/pm/issues/229))
```

**Pull request merged**